CMakeLists.txt

```diff
@@ -9,14 +9,30 @@ ENDIF()
 MESSAGE(STATUS "Build type: ${CMAKE_BUILD_TYPE}")
 FIND_PACKAGE (benchmark QUIET)
 
-SET(COMMON_LIB ev)
+SET(CMAKE_CXX_STANDARD 17)
+SET(CMAKE_C_STANDARD 11)
+
+ADD_LIBRARY(tntcxx INTERFACE)
+ADD_LIBRARY(tntcxx::tntcxx ALIAS tntcxx)
+TARGET_COMPILE_OPTIONS(tntcxx INTERFACE -Wall -Wextra -Werror)
```

Comment:
Compile options should be applied to all targets.

Reply:
Fixed, thanks!